core/src/main/java/org/apache/accumulo/core/client/ClientConfiguration.java

```diff
@@ -656,6 +656,16 @@ public void addConfiguration(Configuration config) {
     super.addConfiguration(config);
   }
 
+  /**
+   * @deprecated since 1.9.2; will be removed in 2.0.0 to eliminate commons config leakage into
```

Comment:
Shouldn't this be `since 1.9.3`?

Reply:
In 1.9.2, we deprecated everything exposed from the superclass... but couldn't document this one in our subclass because the method wasn't available in our build time class path for us to know about it. It would still have been deprecated then, though, if a user had put this newer version on their class path. So, I'm not sure it matters.

Reply:
I have no objection to setting it to 1.9.2.  It is a complex case.

Reply:
FWIW, I have no objection to changing it to 1.9.3 for the same reason :smiley_cat: